src/ui_ng/lib/src/helm-chart/chart-detail/chart-detail-summary.component.scss

```diff
@@ -1,37 +1,68 @@
 .content-wrapper {
-    margin-top:20px;
+    margin-top: 20px;
     padding: 0 0 0 15px;
-
     .md-container {
         border: solid 1px #ddd;
     }
-    
     .summary-container {
-        padding:0;
+        padding: 0;
         table {
             background-color: #F2F2F2;
             margin-top: 0.5rem;
         }
-    
         .content-group {
             margin-bottom: 30px;
         }
-        
         .content-icon {
             margin-right: 6px;
         }
-    
         .text-wrapper {
             word-break: break-all;
         }
     }
-
     .cmd-title {
         white-space: nowrap;
         vertical-align: middle;
     }
-
     .cmd-content {
         width: 100%;
     }
+}
```

Comment:
scss using a nested structure to avoid duplications in scss for tags. So this css should be refactor something like this:

:ng-deep  {
     
    table {
        display: block;
        width: 100%;
        overflow: auto;
        padding: 0;
        border-spacing: 0;
        border-collapse: collapse;
        margin-bottom: 16px;
        td,
        th {
            padding: 6px 13px;
            border: 1px solid #ddd;

            &[align="left"] {
                text-align: left;
            }

            &[align="center"] {
                text-align: left;
            }
            &[align="right"] {
                text-align: left;
            }

        }
        tr {
            &:nth-child(2n) {
                background-color: #F2F2F2;
            }
            
        }
    }
} 

This is just an example, I did not add all the things, please check it .

Reply:
even something can be improve, like this:

@mixin align-text-mixin ($values...) {
    @each $var in $values {
        &[align=$var] {
            text-align: $var;
        }
    }
}

:ng-deep  {
     
    table {
        display: block;
        width: 100%;
        overflow: auto;
        padding: 0;
        border-spacing: 0;
        border-collapse: collapse;
        margin-bottom: 16px;
        td,
        th {
            padding: 6px 13px;
            border: 1px solid #ddd;

           @include align-text-mixin(left, right,center);

        }
        tr {
            &:nth-child(2n) {
                background-color: #F2F2F2;
            }
            
        }
    }
}